PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -572,7 +572,7 @@ intptr_t CALLBACK Shortcut::run_dlgProc(UINT Message, WPARAM wParam, LPARAM lPar
 				}
 
 				if (_keyCombo._key == namedKeyArray[i].id)
-					iFound = static_cast<int32_t>(oemVkUsedIDs.size()-1);	// -1 because it's already been added, so the most recent index of oemVkUsedIDs is 1 less than the length
+					iFound = static_cast<int32_t>(oemVkUsedIDs.size())-1;	// -1 because it's already been added, so the most recent index of oemVkUsedIDs is 1 less than the length

```

Comment:
It's always better to keep the code readability: Write `static_cast<int32_t>(oemVkUsedIDs.size()) - 1` instead of `static_cast<int32_t>(oemVkUsedIDs.size())-1`. But it's OK for this time.